user-registration-app/src/app/views/live-view/live-view.component.html

```diff
@@ -305,279 +303,427 @@
+                  the ethical impacts our technology may have on us? Listen as Dr.
+                  Droege presents a new perspective on accounting for these ethical
+                  impacts, as hackers are working to push society’s boundaries.
                 </p>
-                <p>Uses: Anaconda</p>
               </div>
             </div>
             <div class="card beginner">
-              <div class="card-image waves-effect waves-block waves-light">
-                <img class="activator" src="../../../assets/images/workshop_leaders/ryan.png">
-                <a class="btn halfway-fab waves-effect waves-light red activator">click to learn more</a>
+              <div class="card-image">
+                <img class="activator" src="../../../assets/images/workshop_leaders/Vanguard.svg">
+                <a class="waves-effect waves-light activator red btn">click to learn more</a>
               </div>
               <div class="card-content pt-5">
-                <span class="card-title activator" style="font-weight:bold;">Web Design Using CSS/HTML</span>
-                <div style="font-weight:bold;">Ryan O’Neill, Co-Executive Director of HackPSU</div>
+                <span class="card-title activator" style="font-weight:bold;">Building and Deploying Your First Cloud Native Microservice</span>
+                <div style="font-weight:bold;">Vanguard Team</div>
+                <div style="font-weight:bold;">Location: 125 Business Building (Workshop Room 2)</div>
+                <div style="font-weight:bold;">Time: 6:00 PM - 6:45 PM Saturday</div>
+              </div>
+              <div class="card-reveal">
+                <span class="card-title">Building and Deploying Your First Cloud Native Microservice<i
+                  class="material-icons right">close</i></span>
+                <p>Ever heard the term monolith or microservice? How about distributed system,
+                  decoupling, or modularization? All are part of the discussion about the ongoing
+                  transformation in tech, towards smaller and more flexible systems. In this workshop,
+                  we will build a microservice in Node.js that is simple and scalable. We will then deploy
+                  the microservice to the cloud, utilizing Heroku. Prior experience with Javascript will
+                  be helpful but not required.
+                </p>
+              </div>
+            </div>
+            <div class="card beginner">
+              <div class="card-image">
+                <img class="activator" src="../../../assets/images/workshop_leaders/Emma_Neil.png">
+                <a class="waves-effect waves-light activator red btn">click to learn more</a>
+              </div>
+              <div class="card-content pt-5">
+                <span class="card-title activator" style="font-weight:bold;">3D Printing Overview</span>
+                <div style="font-weight:bold;">Emma Neill & Alex Ambruso</div>
```

Comment:
Is it Emma Neil*?

Reply:
Nope. One of the few that is not a typo lmao, just checked the workshop stuff from clay.